src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,103 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final double CAPACITY_FACTOR = 0.75;
+    private Node<K, V>[] entryset = new Node[16];
```

Comment:
extract magic number to a constant. and initialise entryset in constructor

Reply:
magic number - ok.
initialise in constructor, I don't agree. It will be initialised when new MyHashMap() is called, just before empty constructor is called, right?